File: src/main/java/demo/Collections.java
// ArrayList (ordered, allows duplicates)
List<String> names = new ArrayList<>();
names.add("Alice");
names.add("Bob");
names.add("Charlie");

// HashSet (unordered, no duplicates)
Set<Integer> numbers = new HashSet<>();
numbers.add(1);
numbers.add(2);
numbers.add(1); // Duplicate not added

// Looping through elements
for (String name : names) {
  System.out.println(name);
}

File: src/main/kotlin/demo/Collections.kt
// Mutable list (ordered, allows duplicates)
val names: MutableList<String> = mutableListOf("Alice", "Bob", "Charlie")
names.add("David")

// Set (unordered, no duplicates)
val numbers: Set<Int> = setOf(1, 2, 1) // Duplicate is ignored

// Using lambda expression for iteration
names.forEach { name -> println(name) }